### ISSUE TITLE
isInGame() always true 

### ISSUE THREAD

**Issue by rawora** [meraki-analytics/orianna#88]
Hey,
isInGame() seems to always return true, even if the summoner is not in a game.

I made an if-statement:
`  if (summoner.isInGame()) {
			    	 System.out.println("is");
				} else {
					System.out.println("is not");
				}`
and no matter which summoner I choose, it says they all are in a game.

Thank you

**Comment by BillyD73** [comment]
As a workaround have you tried an alternative method such as: 

```
if (Orianna.currentMatchForSummoner(summoner).get() ! = null)
{
}
```

I'll soon be working on adding current games to my app too so I'll keep you posted, that's the code I currently wrote but it needs testing!

**Comment by BillyD73** [comment]
Deleted my previous response before it was verified. After trying and verifying on the Discord the correct way to do this is:

`Orianna.currentMatchForSummoner(summoner).get();`

If you get a 403 "Not found" response then the Summoner isn't in a game, else if the line above returns not null they are :)

**Comment by robrua** [comment]
Hey, sorry for the delay. This should be fixed with ce42468 and should now be in the latest SNAPSHOT. Will be included in next release.
